historical/rollup.config.js

```diff
@@ -47,11 +47,7 @@ export default {
     //   runtime: require('node-sass'),
     //   options: {
     //     sourceMap: development,
-    //     outputStyle: development ? 'nested' : 'compressed',
-    //     includePaths: [
-    //       './node_modules/bulma',
-    //       './node_modules/@fortawesome/fontawesome-free/scss'
-    //     ]
```

Comment:
Please do not modify files in `historical/`, that defeats the purpose of having them around :P